app/src/main/java/com/teambeme/beme/detail/adapter/ReplyAdapter.kt

```diff
@@ -50,6 +50,7 @@ class ReplyAdapter(private val context: Context, private val viewModel: DetailVi
         val open_btn: TextView = binding.txtReplyparentOpen
         val child_rcv: RecyclerView = binding.rcvReplyparentChild
         val dot: ImageView = binding.imgReplyparentDot3
+        val addReply: TextView = binding.txtReplyparentAdd
```

Comment:
- 뷰를 변수로 받아온 이유가 있을까
- 그리고 변수면 camelCase!
- dot이라고만 하면 저게 ImageView인지 어떻게 알까? 
```suggestion
        val textReplyParentOpen: TextView = binding.txtReplyparentOpen
        val childReplyRecyclerView: RecyclerView = binding.rcvReplyparentChild
        val imgReplyButton: ImageView = binding.imgReplyparentDot3
        val textReplyAdd: TextView = binding.txtReplyparentAdd
```

그냥 생각난대로 지은거라 이대로 바꾸라는 건 아닌데, 가독성 있는 변수명을 지어줘! 형이 한 번 만든 변수 이름은 이 프로젝트 하는 동안 계속 보게될거야